src/components/sbb-journey-summary/sbb-journey-summary.tsx

```diff
@@ -0,0 +1,157 @@
+          </time>
+          {this._formatTime(duration)}
+        </span>
+      );
+  }
+
+  private _renderJourneyVias(vias: string[]): JSX.Element {
+    const slicedVias = vias.slice(0, 4);
+    return (
+      <div class="journey-summary__via-block">
+        <span class="journey-summary__via-text">Via</span>
+        <ul class="journey-summary__vias">
+          {slicedVias.map((via, index) => (
+            <li class="journey-summary__via">
+              {via}
+              {index !== slicedVias.length - 1 && <span>, </span>}
+            </li>
+          ))}
+        </ul>
+      </div>
+    );
+  }
+
+  /** renders the optional walktimes, the icon is displayed either before or after the time*/
+  private _renderWalkTime(iconBefore: boolean, duration: number): JSX.Element {
+    return (
+      <span class="journey-summary__walktime">
+        {iconBefore && <sbb-icon name="walk-small"></sbb-icon>}
+        <time dateTime={duration.toString()}>
+          {duration}
+          <span aria-hidden="true">'</span>
+        </time>
+        <span class="screenreaderonly">
+          {iconBefore
+            ? i18nWalkingDistanceDeparture[this._currentLanguage]
+            : i18nWalkingDistanceArrival[this._currentLanguage]}
+        </span>
+        {!iconBefore && <sbb-icon name="walk-small"></sbb-icon>}
+      </span>
+    );
+  }
+
+  public render(): JSX.Element {
+    const { vias, origin, destination } = this.summaryConfig || {};
+    const departureTime: Date = parseISO(this.summaryConfig?.departure?.time);
+    const arrivalTime: Date = parseISO(this.summaryConfig?.arrival?.time);
+
+    return (
+      <div class="journey-summary">
+        {origin && (
+          <sbb-journey-header origin={origin} destination={destination}></sbb-journey-header>
+        )}
+        {vias && this._renderJourneyVias(vias)}
+        <div class="journey-summary__body">
+          {this._renderJourneyStart(departureTime, arrivalTime)}
+          <div class="journey-summary__transportation-details">
+            <span class="screenreaderonly">{i18nDeparture[this._currentLanguage]}</span>
+            {this.summaryConfig?.departureWalk &&
+              this._renderWalkTime(true, this.summaryConfig?.departureWalk)}
+            {isValid(departureTime) && (
+              <time class="journey-summary__time">{format(departureTime, 'HH:mm')}</time>
+            )}
+            <div class="journey-summary__pearlchain">
+              <sbb-pearl-chain legs={this.summaryConfig?.legs} />
+            </div>
+            {isValid(arrivalTime) && (
+              <time class="journey-summary__time">{format(arrivalTime, 'HH:mm')}</time>
+            )}
+            <span class="screenreaderonly">{i18nArrival[this._currentLanguage]}</span>
+            {this.summaryConfig?.arrivalWalk &&
+              this._renderWalkTime(false, this.summaryConfig?.arrivalWalk)}
+          </div>
+          <div class="journey-summary__slot">
+            <slot />
```

Comment:
Excellent use of a slot! <3 Any chance to render the wrapping div conditionally (dep. on existance of slot)?